### PR TITLE
Clarify storybook bug

### PR DESCRIPTION
**Pull request by luke-hill**
Ronseal PR

### CODE REVIEW

File: testing/features/components/navigation.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# NB: Due to a Storybook issue, these tests won't run on Old Safari / iOS
```
Comment (davidrapson) [inline testing/features/components/navigation.feature:1]: What's the actual issue? Longer term we could probably start running these tests against the `demo/` app which is a much simpler and more realistic environment than storybook but we should chat about that first 🤓

**Review by davidrapson — APPROVED** [review]
👍 


(Small aside, I'm always unsure what you mean by "Ronseal PR", if possible adding a real description even if the change _feels_ obvious is super helpful. Thanks 🙏)

File: testing/features/components/navigation.feature (hunk 1 above)
Comment (luke-hill) [reply testing/features/components/navigation.feature:1]: Go for it. Happy to let you guys decide on this. You probably know better.

P.S. Showing my age but: https://www.youtube.com/watch?v=OkGaq9xiQZY

Comment (davidrapson) [reply testing/features/components/navigation.feature:1]: Hah, I see. Didn't make that connection.

Yeah, that probably makes sense it'd be some compatibility thing. Storybook doesn't have a very good cross-browser story. No need to switch now, but as we've migrate more components into the Rails engine there will be a pretty decent test environment within the demo app. Something to come back to.

Comment (luke-hill) [reply testing/features/components/navigation.feature:1]: I've popped a new Phase I ticket in the backlog for this. Happy to tackle this whenever you feel is appropriate. I don't envisage it being a lot of work our end to migrate the tests (Probably just altering some page urls / ruby setup code).

**Pull request merged**